.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Bug report
+about: Create a report and help us fix a problem
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- 
+    Thank you for using DartPad!
+
+    If you've run into a problem while trying to edit or run code in DartPad,
+    we'd like to help. By providing as much information as possible about
+    your browser, machine, and experience, you can help us identify 
+    problems as quickly as possible.
+
+    Please looks through the questions below and answer as many as you can.
```

Comment:
We could also add a link to https://dart.dev/tools/dartpad/troubleshoot

Reply:
Done.